src/modules/Channel/context/ChannelProvider.tsx

```diff
@@ -140,7 +140,7 @@ interface ChannelProviderInterface extends ChannelContextProps, MessageStoreInte
   isScrolled?: boolean;
   setIsScrolled?: React.Dispatch<React.SetStateAction<boolean>>;
   messageActionTypes: typeof channelActions;
-  messagesDispatcher: CustomUseReducerDispatcher;
+  messagesDispatcher: React.Dispatch<ChannelActionTypes>;
```

Comment:
don't we need to update index.d.ts file accordingly?

Reply:
Yes, we will soon generate type definition through ts source code, so I don't think we have to change it.